extensions/brew/src/utils.ts

```diff
@@ -66,9 +66,59 @@ export async function fetchRemote<T>(remote: Remote<T>): Promise<T[]> {
 
 /// Toast
 
+interface ActionToastOptions {
+  title: string;
+  message?: string;
+  cancelable: boolean;
+}
+
+export function showActionToast(actionOptions: ActionToastOptions): AbortController | undefined {
+  const options: ToastOptions = {
+    style: ToastStyle.Animated,
+    title: actionOptions.title,
+    message: actionOptions.message
+  };
+
+  let toast: Toast | undefined;
```

Comment:
Looks like the linter doesn't like this line and it makes the CI fail. Is there any way to please the linter here? CC @mathieudutour and @grzegorzkrukowski